src/main/webapp/script.js

```diff
@@ -33,7 +34,13 @@ function displayLocation(position) {
 }
 
 function displayError() {
-  // TODO: Write error to user interface
   console.log('Geolocation not enabled');
   alert(alertMessage);
+}
+
+function getZip() {
+  var zip = document.getElementById('zipCode').value;
+  var xhttp = new XMLHttpRequest();
```

Comment:
Reminder, we decided to prefer const or let.

Reply:
Ah, yes gotta love Javascript!